portfolio/src/main/webapp/budget.html

```diff
@@ -7,7 +7,7 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <h1>Budget Page</h1>
+    <h1>Budget page</h1>
```

Comment:
Please change to p tags.

```suggestion
    <p>Budget page</p>
```